File: water/src/main/resources/static/js/component/invoicepublic/InvoiceInvalid.js
/**
	 
				FPZL	发票种类	字符	2	是	固定值：
								0：专用发票 
								2：普通发票
								12：机动车票
								51：电子发票
				FPHM	发票号码	字符	20	是	
				FPDM	发票代码	字符	20	是
			
			
			private String FPZL;
			private String FPHM;
			private String FPDM;
 */

function InvoiceInvalid(){
	this.FPZL="0"; //发票种类
	this.FPHM=""; //发票号码
	this.FPDM="";  //发票代码
}

//-------------getter and setter---------------
//发票类种
InvoiceInvalid.prototype.getFPZL=function(){
	return this.FPZL;
}
InvoiceInvalid.prototype.setFPZL=function(fpzl){
	this.FPZL=fpzl;
}
//FPHM 发票号码
InvoiceInvalid.prototype.getFPHM=function(){
	return this.FPHM;
}
InvoiceInvalid.prototype.setFPHM=function(fphm){
	this.FPHM=fphm;
}

//FPDM 发票代码
InvoiceInvalid.prototype.getFPDM=function(){
	return this.FPDM;
}
InvoiceInvalid.prototype.setFPDM=function(fpdm){
	this.FPDM=fpdm;
}

File: water/src/main/java/com/learnbind/ai/tax/packet/request/InvoiceInvalid.java
package com.learnbind.ai.tax.packet.request;

/**
 * Copyright (c) 2019 by Hz
 * 
 * @Package com.learnbind.ai.tax.packet.request
 *
 * @Title: InvoiceInvalid.java
 * @Description: 发票作废
 *
 * @author lenovo
 * @date 2019年10月17日 上午9:05:20
 * @version V1.0 
 *
 */
public class InvoiceInvalid {
	/*
	 
		FPZL	发票种类	字符	2	是	固定值：
						0：专用发票 
						2：普通发票
						12：机动车票
						51：电子发票
		FPHM	发票号码	字符	20	是	
		FPDM	发票代码	字符	20	是
	 */
	
	
	private String FPZL;
	private String FPHM;
	private String FPDM;
	
	public String getFPZL() {
		return FPZL;
	}
	public void setFPZL(String fPZL) {
		FPZL = fPZL;
	}
	public String getFPHM() {
		return FPHM;
	}
	public void setFPHM(String fPHM) {
		FPHM = fPHM;
	}
	public String getFPDM() {
		return FPDM;
	}
	public void setFPDM(String fPDM) {
		FPDM = fPDM;
	}
	
	

}
